balancer/frontend-v2
# Add 90MASQ-10wstETH

add new weighted pool for 90MASQ-10wstETH to allowlist on frontend-v2

# Description

Kindly requesting a new pool to be added to the frontend-v2 allowlist - 90MASQ-10wstETH

I have had comms with solarcurve before, to assist in adding our MASQ token (back in April), so we do already have a pool on allowlist which is 70MASQ-15WMATIC-15USDC, and MASQ is already on tokenlist

## Type of change

- [x] Other - adding pool to allowlist

## How should this be tested?

Run any automated tests to verify the pool ID relates correctly to [this pool](https://app.balancer.fi/#/polygon/pool/0x625bfdd5052c27c45c8482efb26990af66aea94a000100000000000000000a90)


## Visual context

Please provide any relevant visual context for UI changes or additions. This could be static screenshots or a loom screencast.

## Checklist:
